app/mailers/application_mailer.rb

```diff
@@ -1,2 +1,2 @@
-class ApplicationMailer < ActionMailer::Base
+class ApplicationMailer < Mail::Notify::Mailer
```

Comment:
I'm guessing this is so that RequestNotifier also has access to the Notify methods?

Reply:
This is actually more Rails convention that your app has a single `ApplicationMailer` and then the custom mailers (in this case `Notifier`) inherits from `ApplicationMailer`. Since we're only going to be using Notify from now on I figure that `ApplicationMailer` can inherit from `Mail::Notify::Mailer` (which happens to also inherit from `ActionMailer::Base`).

Reply:
Oh I see — overriding in `ApplicationMailer` (if necessary) would be harder otherwise. Will bear this in mind when migrating the other apps. 🙂